index.html

```diff
@@ -48,7 +50,200 @@ <h1 class="h1">I’m Patrick Ndimukulaga Glad to see you!</h1>
           </li>
         </ul>
       </div>
+      </div>
+    </section>
+    
+    <div class="simi">
+      <div class="title grid">
+       
+        <div>
+          <img src="./images/Snapshoot_Portfolio.png" alt="portfolio"/>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_58](https://user-images.githubusercontent.com/69638013/165104230-03279a58-262c-4449-8a5f-b88600b145b5.png)  | ![Screenshot_57](https://user-images.githubusercontent.com/69638013/165104192-1d5c92f8-6254-48cc-8b82-7b85a3a45376.png)

- [ ] Kindly adjust image in each card to have the same padding like the Figma design, please make adjustments for all images in this section 👍🏿